archlinuxcn/archlinuxcn-mirrorlist/lilac.yaml

```diff
@@ -4,3 +4,4 @@ maintainers:
 update_on:
   - source: github
     github: archlinuxcn/mirrorlist-repo
+    path: archlinuxcn-mirrorlist
```

Comment:
How can I write rules to scan only the files in the root directory and exclude the subdirectories?